shared/src/common/enums/validation/validation.ts

```diff
@@ -1,6 +1,8 @@
 export {
   EAMWorkerValidationMessage,
   EAMWorkerValidationRule,
+  EAMWorkerValidationMessageBackend,
```

Comment:
you don't need to create new enums
you need to add messages and rules to `EAMWorkerValidationMessage`, `EAMWorkerValidationRule`